packages/contracts-periphery/contracts/universal/op-nft/Optimist.sol

```diff
@@ -90,28 +85,22 @@ contract Optimist is ERC721BurnableUpgradeable, Semver {
     }
 
     /**
-     * @notice Returns the URI for the token metadata.
-     * @dev The token URI will be stored at baseURI + '/' + tokenId + .json
-     * @param tokenId The token ID to query.
-     * @return The URI for the given token ID.
+     * @notice Returns the token URI for a given token by ID
+     *
+     * @param _tokenId Token ID to query.
+
+     * @return Token URI for the given token by ID.
      */
-    function tokenURI(uint256 tokenId) public view virtual override returns (string memory) {
-        return
-            string(
-                abi.encodePacked(
-                    baseURI(),
-                    "/",
-                    // convert tokenId to hex string formatted like an address (20)
-                    Strings.toHexString(tokenId, 20),
-                    ".json"
-                )
-            );
+    function tokenURI(uint256 _tokenId) public view virtual override returns (string memory) {
+        return string(abi.encodePacked(baseURI(), "/", Strings.toHexString(_tokenId, 20), ".json"));
```

Comment:
how is this change not breaking linter?

Reply:
This change was made by the linter

Reply:
oh I see now because you removed the comment!  I thought prettier randomly changed it's mind on how this is formatted

Reply:
```suggestion
        // convert tokenId to hex string formatted like an address (20)
        return string(abi.encodePacked(baseURI(), "/", Strings.toHexString(_tokenId, 20), ".json"));
```
I think we need this comment actually.   THe original contracts I got handed had a bug here so it isn't 100% intuitive